data-products/src/recommendation_api/sql/scheduled_syndicated.sql

```diff
@@ -1,11 +1,13 @@
 SELECT
-    APPROVED_CORPUS_ITEM_EXTERNAL_ID as "ID",
-    TOPIC as "TOPIC",
-    PUBLISHER AS "PUBLISHER"
-FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS"
-WHERE SCHEDULED_SURFACE_ID = %(SCHEDULED_SURFACE_ID)s
-AND approved_corpus_item_external_id NOT IN ('904bc245-54de-412a-a8af-c13eb0a7c15c') -- See #incidents on 2023-03-21 and DENG-4478
-AND IS_SYNDICATED
-AND SCHEDULED_CORPUS_ITEM_SCHEDULED_AT BETWEEN DATEADD(day, %(MAX_AGE_DAYS)s, CURRENT_DATE) AND CURRENT_DATE
-QUALIFY row_number() OVER (PARTITION BY APPROVED_CORPUS_ITEM_EXTERNAL_ID ORDER BY SCHEDULED_CORPUS_ITEM_SCHEDULED_AT DESC) = 1
-ORDER BY SCHEDULED_CORPUS_ITEM_SCHEDULED_AT DESC
+    s.APPROVED_CORPUS_ITEM_EXTERNAL_ID as "ID",
+    i.TOPIC as "TOPIC",
+    i.PUBLISHER AS "PUBLISHER"
+FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS" AS s
+LEFT JOIN "ANALYTICS"."DBT"."CORPUS_ITEMS_UPDATED" AS i
```

Comment:
fyi: does not make a big diff if its an outer join vs. an inner join here. the `TOPIC IS NOT NULL` will force an inner join anyway.

Also, want to take this opportunity to answer your previous question of why I chose -3 hours for `corpus_item_schedules_updated`. Its because _sometimes_ a newly added corpus item (although created when a schedule gets created) could come in with a short delay (Snowplow latency). The -3 could address that situation by reprocessing the schedules again and backfilling with the previously missing metadata.

```suggestion
JOIN "ANALYTICS"."DBT"."CORPUS_ITEMS_UPDATED" AS i
```

Reply:
Why 3 hours?

Reply:
this was merged prior to the join suggestion above @gkatre so if it doesn't matter then i'll leave it as is

Reply:
@mmiermans you seem to have asked the same question on slack. Replied [there](https://mozilla.slack.com/archives/C05EVRU1U1X/p1726263224023009?thread_ts=1725910997.253949&cid=C05EVRU1U1X)